api/user_buckets_lifecycle.go

```diff
@@ -102,7 +102,12 @@ func getBucketLifecycle(ctx context.Context, client MinioClient, bucketName stri
 				Value: tagData.Value,
 			})
 		}
-
+		if len(rule.RuleFilter.Tag.Key) > 0 {
```

Comment:
I think this is more idiomatic Go:
```suggestion
		if rule.RuleFilter.Tag.Key != "" {
```